tests/data/dataconverter/readers/mpes/Ref_nexus_mpes.log

```diff
@@ -702,25 +665,15 @@ DEBUG - value: 0.11
 DEBUG - classpath: ['NXentry', 'NXinstrument', 'NXbeam', 'NX_NUMBER']
 DEBUG - classes:
 NXmpes.nxdl.xml:/ENTRY/INSTRUMENT/BEAM/incident_energy_spread
-NXbeam.nxdl.xml:/incident_energy_spread
 DEBUG - <<RECOMMENDED>>
 DEBUG - documentation (NXmpes.nxdl.xml:/ENTRY/INSTRUMENT/BEAM/incident_energy_spread):
 DEBUG - 
-DEBUG - documentation (NXbeam.nxdl.xml:/incident_energy_spread):
```

Comment:
Are these missing because base class modifications have been removed, like incident_energy_spread from NXbeam?
They will come back when merging Florian's corrections...

Reply:
You are right, this work on this branch happened during an interval in time when these already discussed about mpes base class fields were already removed but development for Leoben had to move on and thus I had to adapt to the situation of using a commit of nxdefs during this pullback interval. It is also my assumption that when Florians mpes base class back branch is merged into nxdefs we can and should update the nxdefs on the fairmat anyway and this will demand that I test again then all examples if they work with the current combination of nxdefs + up-to-date and update-north-markus merged into fairmat etc. happy to do this after the merge of both according to how tomorrow's meeting these testing tasks I think is what is needed and what I meant with I will have time this week to prioritize it.